internal/strict/controls/deprecated_flag_name.go

```diff
@@ -40,8 +41,8 @@ func NewDeprecatedMovedFlagName(deprecatedFlag, newFlag cli.Flag, commandName st
 			Name:        deprecatedName,
 			Description: "replaced with: " + newName,
 		},
-		ErrorFmt:       "The `--%s` global flag is no longer supported. Use `--%s` instead.",
-		WarningFmt:     "The `--%s` global flag is moved to `" + commandName + "` command and will be removed from the global flags in a future version. Use `--%s` instead.",
+		ErrorFmt:       "The `--%s` global flag is no longer supported. Use `%s` instead.",
+		WarningFmt:     "The `--%s` global flag is moved to `" + commandName + "` command and will be removed from the global flags in a future version. Use `%s` instead.",
```

Comment:
```suggestion
		WarningFmt:     "The `--%s` global flag has moved to the `" + commandName + "` command and will be removed from global flags in a future version of Terragrunt. Use `%s` instead.",
```

These initially confused me. It would help to add a comment above this that the second placeholder has both the command and the flag, and that's why it doesn't have the dashes.